torchtune/utils/quantization.py

```diff
@@ -0,0 +1,47 @@
+from typing import Any
```

Comment:
We should also add some tests to make sure the model is as expected after quantization. This will help catch any breakages in APIs/behaviors if torchao changes.

@rohan-varma any ideas around testing?

Reply:
@kartikayk I changed these to just use classes directly and they will be tested in torchao, is that OK?